voila/handler.py

```diff
@@ -24,29 +25,20 @@
 from .utils import ENV_VARIABLE
 
 
-class VoilaHandler(JupyterHandler):
-    def initialize(self, **kwargs):
-        self.notebook_path = kwargs.pop('notebook_path', [])  # should it be []
-        self.template_paths = kwargs.pop('template_paths', [])
-        self.traitlet_config = kwargs.pop('config', None)
-        self.voila_configuration = kwargs['voila_configuration']
-        # we want to avoid starting multiple kernels due to template mistakes
-        self.kernel_started = False
+async def _get(self, path=None):
```

Comment:
I think we might need docstring and typing for this function because using `self` as a function parameter is a little bit confusing.